ding/world_model/model/ensemble.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ding.torch_utils import Swish
+
+
+class StandardScaler(nn.Module):
+
+    def __init__(self, input_size):
+        super(StandardScaler, self).__init__()
+        self.register_buffer('std', torch.ones(1, input_size))
+        self.register_buffer('mu', torch.zeros(1, input_size))
+
+    def fit(self, data):
+        std, mu = torch.std_mean(data, dim=0, keepdim=True)
+        std[std < 1e-12] = 1
+        self.std.data.mul_(0.0).add_(std)
+        self.mu.data.mul_(0.0).add_(mu)
+
+    def transform(self, data):
+        return (data - self.mu) / self.std
+
+    def inverse_transform(self, data):
```

Comment:
This part is directly copied from the original MBPO codebase in DIengine, which is directly copied from https://github.com/Xingyu-Lin/mbpo_pytorch/blob/main/model.py#L64.
This model is specific to `MBPO` (and `DDPPO`), whose API is not as important as other abstract classes or modules designed for reuse. And this will potentially go through further modifications. Therefore, I may not plan to add comments in this file for this PR. Correct me if I am wrong.

Reply:
linting finished